server/gfmd/user.c

```diff
@@ -47,6 +52,8 @@ struct user {
 	int needs_chroot;
 	struct tenant *tenant;
 	char *name_in_tenant;
+
+	struct user_auth *user_auths;
```

Comment:
We would like to consider the implementation here later, so please remove the addition of user_auths